### PR TITLE
Changed build tool to hemtt

### PR DESCRIPTION
**Pull request by GermanHydrogen**
Adapted mod structure to the build tool *HEMTT* (https://github.com/BrettMayson/HEMTT)

MP testing TODO:

- [x] ACRE
- [x] antiCheat
- [x] buildPop
- [x] ViewDistance
- [x] EasySpawn
- [x] HABO
- [x] initBlackScreen
- [ ] pvp?
- [x] teleport
- [x] zoneRestrictions

### CODE REVIEW

File: addons/main/3den/feature_persistence.hpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿class RR_commons_persistence {
+class RR_commons_persistence {
```
Comment (TAINCER) [inline addons/main/3den/feature_persistence.hpp:1]: What changed?

Comment (GermanHydrogen) [reply addons/main/3den/feature_persistence.hpp:1]: Diff prob. due to prev. falty file encoding

File: addons/main/functions.h [hunk 2]
```diff
@@ -37,22 +37,22 @@ DEC(acre,acreInit)
 /* ACRE Babel Setup */
 DEC(acre,babelSetup)
 
-/* ACRE Funkger�tvergabe */
```
[the remark this answers is not in the document]
Comment (GermanHydrogen) [reply addons/main/functions.h:40]: Diff prob. due to prev. falty file encoding

File: addons/main/3den/feature_zoneRestriction.hpp [hunk 3]
```diff
@@ -1,7 +1,9 @@
-﻿class RR_commons_zoneRestriction {
+class RR_commons_zoneRestriction {
```
[the remark this answers is not in the document]
Comment (TAINCER) [reply addons/main/3den/feature_zoneRestriction.hpp:1]: Please remove this one and `functions.h` if nothing changed

**Pull request merged**